docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -119,3 +197,41 @@ The important part is having a WordPress site installed, and know where and how
 Alternative editors include [Sublime Text](https://www.sublimetext.com/) that is also available across platforms, though is a commercial product; or other free alternatives include [Vim](https://www.vim.org/), [Atom](https://atom.io/), and [Notepad++](https://notepad-plus-plus.org/) all support standard JavaScript style development.
 
 You can use any editor you're comfortable with, it is more a personal preference. The development setup for WordPress block editor is a common JavaScript environment and most editors have plugins and suppport. The key is having a way to open, edit, and save text files.
+
+## Uninstall - Start Over
+
+Here are a few instructions if you need to start over, or want to remove what was installed.
+
+### Local Environment
+
+* You can remove the local environment data for a specific project:
+
+```
+wp-env destroy
+```
+
+* To completely uninstall wp-env tool:
+
+```
+npm -g uninstall @wordpress/env
+```
+
```

Comment:
Testing:

Btw I got a patch notice in terminal.

```
paaljoachimromdahl@Paals-MacBook-Pro gutenberg % npm -g uninstall @wordpress/env
removed 274 packages in 2.881s
```
   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.14.6 → 6.14.8       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.8   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯

Reply:
Unrelated.